patches/itk/default.nix

```diff
@@ -13,17 +13,11 @@ stdenv.mkDerivation rec {
     "-DBUILD_EXAMPLES=OFF"
     "-DBUILD_SHARED_LIBS=ON"
 
-    # disable default modules
-    "-DITK_BUILD_DEFAULT_MODULES=OFF"
-     "-DModule_ITKImageGrid=ON"
-#    "-DModule_ITKIOMINC=ON"
-#    "-DModule_ITKVtkGlue=ON"
-#    "-DModule_ITKReview=ON"
-
-
+    # enable default modules
+    "-DITK_BUILD_DEFAULT_MODULES=ON"
 
     # disable bundling
-    "-DITK_USE_SYSTEM_HDF5=ON"
+    "-DITK_USE_SYSTEM_HDF5=OFF"
```

Comment:
I read this as:

* we add hdf5 to the dependencies
* tell itk to use whatever it bundles internally rather than the nix installed hdf5

is this the intent? Does the build log indicate that the right hdf5 is picked up?

Reply:
@matz-e thanks, it is fixed. on top of that it is hdf5-cpp package which was needed